test/over_react/component_declaration/builder_integration_tests/abstract_accessor_integration_test.dart

```diff
@@ -150,16 +153,16 @@ main() {
 abstract class _$TestAbstractProps extends UiProps {
   String stringProp;
   dynamic dynamicProp;
-  var untypedProp;
+  dynamic untypedProp;
```

Comment:
This needs to stay as `var` for the purposes of this test.
```suggestion
  // ignore: prefer_typing_uninitialized_variables
  var untypedProp;
```